source/index.js

```diff
@@ -9,6 +9,7 @@ const {
 } = require('dns');
 const {promisify} = require('util');
 const os = require('os');
+const {setTimeout} = require('timers');
```

Comment:
Is there an actual need to require `timers`? Isn't `if (timeout.unref) { timeout.unref(); }` enough?

I am not experienced with Electron nor I'm going to, so I really don't know.

Reply:
According to https://github.com/electron/electron/issues/21162, if we want `unref` to happen, that's necessary. I'm fine either way.

Reply:
It's a conditional in execa: https://github.com/sindresorhus/execa/blob/120230cade59099214905ac2a9136e406c0b6f3a/lib/kill.js#L24-L30

Reply:
> if we want unref to happen, that's necessary. I'm fine either way.

But does it have any effect?

Reply:
I'm guessing it does. Electron has node integration in the renderer process so instead of the blink setTimeout, the node setTimeout is used when you import timers.

Reply:
We can do both as well. It's not just Electron where .unref isn't a function. https://github.com/sindresorhus/get-port/pull/40/files

Reply:
Okay, but `.unref()` makes the process exit if all other event loop tasks are finished. And since it's running in a renderer process, the process itself will run as long as the user keeps it open. So I don't think `.unref()` will have any effect here. If the end user decides to close the electron app, the timeouts are destroyed anyway.

Reply:
You can remove this line if you decide to go with `if`.